guide/13-managing-arcgis-applications/enterprise-sites-guide.ipynb

```diff
@@ -0,0 +1,578 @@
+{
```

Comment:
<p>The Site Editor provides a user with several capabilities in the form of <strong>cards</strong> to simplify their site building and editing workflows. However, applying a common change to multiple sites and pages in your organization can become a tedious process. &nbsp;Using the site layout editing functionality supported in the Python API, you can successfully apply a common change across all the necessary sites and pages.  &nbsp;In order to programmatically edit a site, we need to take a deeper dive into the data model of a Site/Page item. The site layout has <strong>Sections</strong> that contain <strong>Rows</strong>, and the rows further contain <strong>Cards</strong>.  &nbsp;Sections have configurable attributes, like background color and font-color, and are used to theme a site or page. Rows are the building blocks of your site and are implicitly created when you add a card to a section below another card. Whenever you want to add a card, such as a text card or image card, you must have a row card positioned where you want to add that content. Multiple cards can fit in a row card. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1450/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>